docs/astro/src/content/docs/guide/platforms/embedded.mdx

```diff
@@ -14,15 +15,303 @@ we recommend using a recent desktop operating system and compiler version.
 Slint runs on a variety of embedded Linux platforms. Generally speaking, Slint requires a modern Linux userspace
 with working OpenGL ES 2.0 (or newer) or Vulkan drivers. We've had success running Slint on
 
--   Yocto based distributions. For C++ applications see [meta-slint](https://github.com/slint-ui/meta-slint) for recipes. Rust application work out of the box with Yocto's rust support.
+-   Yocto based distributions. 
 -   BuildRoot based distributions.
--   [TorizonCore](https://www.torizon.io/torizoncore-os).
+-   Torizon.
+
+<Tabs>
+<TabItem label="Yocto Linux">
+For C++ applications see [meta-slint](https://github.com/slint-ui/meta-slint) for recipes. 
+
+Rust application work out of the box with Yocto's rust support.
```

Comment:
```suggestion
Rust application work out of the box with Yocto's Rust support.
```

(but as we've seen with $customer recently, this is an area where we can and should provide additional help; not for this PR of course)